spec/bolt/node/ssh_spec.rb

```diff
@@ -9,21 +9,30 @@
   include BoltSpec::Errors
   include BoltSpec::Files
 
+  def mk_config(conf)
+    Bolt::Config.new(transports: { ssh: conf })
+  end
+
   let(:hostname) { ENV['BOLT_SSH_HOST'] || "localhost" }
   let(:user) { ENV['BOLT_SSH_USER'] || "vagrant" }
   let(:password) { ENV['BOLT_SSH_PASSWORD'] || "vagrant" }
   let(:port) { ENV['BOLT_SSH_PORT'] || 2224 }
```

Comment:
The failing test only passes in Travis if `port` is a string. I'm flummoxed.

Reply:
In Travis, this test relies on checking for `localhost:22` rather than `localhost`, because the Travis image already has `localhost` (specifically, `[1]`) as an entry in `known_hosts`. Net::SSH will only check `localhost:22` if the port is a string; if it's an integer == 22, it checks `localhost`.